definitions/dfe_analytics_dataform.js

```diff
@@ -5,6 +5,7 @@ dfeAnalyticsDataform({
   bqProjectName: "apply-for-qts-in-england",
   bqDatasetName: "events_production",
   bqEventsTableName: "events",
+  urlRegex: "www.apply-for-qts-in-england.education.gov.uk",
```

Comment:
```suggestion
  urlRegex: "apply-for-qts-in-england.education.gov.uk",
```

We don't require the `www` at the beginning of the site, so I don't think we need it here.